app/models/exchange_rates/xe_api.rb

```diff
@@ -0,0 +1,33 @@
+module ExchangeRates
+  class XeApi
+    class XeApiError < StandardError; end
+
+    def initialize(date: Time.zone.today)
+      @date = date.strftime('%Y-%m-%d')
+    end
+
+    def get_all_historic_rates
+      send_request("/v1/historic_rate.json/?from=GBP&date=#{@date}&to=*&amount=1")
```

Comment:
it's a good practice to put  the endpoint `/v1/historic_rate.json/ ` in a constant